v2x_ros_driver/config/wave.json

```diff
@@ -64,7 +64,7 @@
   },
   {
     "name" : "PSM",
-    "psid" : "0082",
+    "psid" : "0027",
```

Comment:
What are these changes exactly?

Reply:
This is a fix for the PSID. Correct is 0x27 from this site:
https://standards.ieee.org/products-programs/regauth/psid/public/